apps/src/templates/instructions/PromptIcon.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import ProtectedStatefulDiv from '../ProtectedStatefulDiv';
+
+/**
+ * Simple component for our icon for hints.
+ */
+const PromptIcon = (props) => (
+  <img src={props.src} id="prompt-icon"/>
+);
+PromptIcon.propTypes = {
+  src: React.PropTypes.string.isRequired
+};
+
+export default PromptIcon;
```

Comment:
This component doesn't provide a ton of value just yet. My next PR is going to focus on the styling of this some (i.e. handling different sized images and what to do with the lighbulb), and at that point I suspect it's function will change some - or it may just go away.